action.yml

```diff
@@ -1,9 +1,9 @@
 name: 'Elm Validate'
 description: 'Runs Elm through elm-format and elm-review then compiles it'
 compile-working-directory:
-    description: 'Working directory of the compile step'
+    description: 'Working directory of linter/review/compile step'
```

Comment:
why not just `working-directory` then? Also, small but "lint"/review/compile makes more sense

Reply:
> why not just working-directory then?

As it's a reserved word, most probably, that will affect the entire action and not just those steps. And we want `yarn install` to run from the project's root.